sklearn/feature_extraction/image.py

```diff
@@ -77,7 +77,8 @@ def _mask_edges_weights(mask, edges, weights=None):
         maxval = edges.max()
     else:
         maxval = 0
-    order = np.searchsorted(np.unique(edges.ravel()), np.arange(maxval + 1))
```

Comment:
Because after line 73, isolated vertices (not making part of any edge) are eliminated for edges array by the mask. Therefore the are left to end when reindexing the remaing vertices (those not eliminated by the mask).